autoware_launch/config/system/diagnostics/planning.yaml

```diff
@@ -88,3 +89,15 @@ units:
     type: diag
     node: planning_validator
     name: trajectory_validation_velocity_deviation
+
+  - path: /autoware/planning/emergency_stop
```

Comment:
I added a comment not to use this path in the TIER IV local autoware_launch repository.